### PR TITLE
Add an ability to process several liability contracts simultaneously by the executor

### PR DESCRIPTION
**Pull request by strdn**
see #24 for details

### CODE REVIEW

File: robonomics_liability/src/robonomics_liability/executor.py [hunk 1]
```diff
@@ -45,68 +45,48 @@ def incoming_liability(msg):
...
+            self.complete.publish(liability_msg)
+            self.result_topic.publish(result)
+            rospy.loginfo('Liability %s finished with %s', liability_msg.address, result.result)
             return FinishLiabilityResponse()
         rospy.Service('finish', FinishLiability, finish_liability)
 
+        def start_liability(msg):
+            try:
+                liability_thread = self.liability_execution_threads[msg.address]
+            except Exception as e:
```
Comment (akru) [inline robonomics_liability/src/robonomics_liability/executor.py]: Is possible to use `KeyError`: the KeyError is raised when accessing an invalid key within a dict.

File: robonomics_liability/src/robonomics_liability/LiabilityExecutionThread.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+            objective_rosbag = get_rosbag_from_file(self.liability.objective)
+            if objective_rosbag is not None:
+                player = Player(objective_rosbag, self.__liability_execution_namespace)
+                player.start()
+                rospy.logdebug('Rosbag player started')
+            else:
+                rospy.logwarn('Skip playing objective using rosbag player in liability %s', self.liability.address)
+
+            while not self.liability_finish:
```
Comment (akru) [inline robonomics_liability/src/robonomics_liability/LiabilityExecutionThread.py]: Is it really needed? In my vision waiting here is legacy from previous version of code.
Only one problem in temporary directory creation, but you can create it once in constructor by https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp and remove after finish.

**Comment by akru** [comment]
Could you rebase commit history, please

**Comment by strdn** [comment]
@akru, please see again

**Pull request merged**